README.md

```diff
@@ -46,6 +47,14 @@
 * [Discord Bot API for processes](https://github.com/kinode-dao/discord_api)
 * [LLM API](https://github.com/kinode-dao/llm), for Groq, OpenAi, as well as llama.cpp.
 
+### Useful Regexes
+* full process addresses: `^([a-z0-9-_.]+)@([a-z0-9\-_]+):([a-z0-9-_]+):([a-z0-9-_.]+)#([a-z]+)$`
+* package IDs: `^([a-z0-9-_]+):([a-z0-9-_.]+)$`
+* process IDs: `^([a-z0-9-_]+):([a-z0-9-_]+):([a-z0-9-_.]+)$`
+* KNS names: `^([a-z0-9-_]+)\.([a-z]+)$`
```

Comment:
I have mixed feelings about `KNS names` splitting the name & TLD while the previous return them together as a single unit. At a minimum we should have a brief description of the expected behavior, maybe an example for each.